scala-xml/src/test/scala/org/http4s/scalaxml/ScalaXmlSuite.scala

```diff
@@ -258,7 +258,7 @@ class ScalaXmlSuite extends CatsEffectSuite with ScalaCheckEffectSuite {
           "iso-2022-kr"
         )
       ),
-      "application/xml; charset=iso-2022kr",
+      "application/xml; charset=iso-2022-kr",
```

Comment:
Ouch, is this a mistake? Maybe we should fix the test in http4s-scala-xml as well.

Reply:
I checked the example in the RFC, it is correct there (and meant to be AFAICT), so I fixed it here. Fixing in http4s-scala-xml is probably a good idea. It won't/shouldn't change the behaviour though as the encoding is specified twice in this example.